### PR TITLE
Add missing parameters for 'blkiotune' virsh cmd

### PR DESCRIPTION
**Pull request by lento-sun**
Signed-off-by: Yi Sun <yisun@redhat.com>

### CODE REVIEW

**Comment by lento-sun** [comment]
related cases also fixed:
https://github.com/autotest/tp-libvirt/pull/2257

File: virttest/virsh.py [hunk 1]
```diff
@@ -4209,7 +4209,9 @@ def vol_upload(name, dfile, options="", **dargs):
     return command(cmd, **dargs)
 
 
-def blkiotune(name, weight=None, device_weights=None, options=None, **dargs):
+def blkiotune(name, weight=None, device_weights=None, device_read_iops_sec=None,
```
Comment (Yingshun) [inline virttest/virsh.py]: please update docstring for the parameters.

File: virttest/virsh.py [hunk 2]
```diff
@@ -4224,6 +4226,14 @@ def blkiotune(name, weight=None, device_weights=None, options=None, **dargs):
         cmd += " --weight %s" % weight
     if device_weights:
         cmd += " --device-weights %s" % device_weights
+    if device_read_iops_sec:
+        cmd += " --device-read-iops-sec %s" % device_read_iops_sec
+    if device_write_iops_sec:
+        cmd += " --device-write-iops-sec %s" % device_write_iops_sec
+    if device_read_bytes_sec:
+        cmd += " --device-read-bytes-se %s" % device_read_bytes_sec
```
Comment (Yingshun) [inline virttest/virsh.py]: typo. s/device-read-bytes-se/device-read-bytes-sec/

File: virttest/virsh.py (hunk 1 above)
Comment (lento-sun) [reply virttest/virsh.py]: added, thx

File: virttest/virsh.py (hunk 2 above)
Comment (lento-sun) [reply virttest/virsh.py]: fixed, thx

File: virttest/virsh.py (hunk 1 above)
Comment (Yingshun) [reply virttest/virsh.py]: @lento-sun, your updates are about params in 'blkdeviotune' but not in 'blkiotune'.

File: virttest/virsh.py [hunk 3]
```diff
@@ -4236,6 +4246,12 @@ def blkdeviotune(name, device=None, options=None,
     Set or get a domain's blkio parameters
```
Comment (Yingshun) [inline virttest/virsh.py]: BTW, I think the description of blkdeviotune() is inappropriate, would you please update it? Thanks!

File: virttest/virsh.py (hunk 1 above)
Comment (lento-sun) [reply virttest/virsh.py]: oops~
updated. The blkdeviotune() has much less available params than virsh api does, need to be updated later. not covered here in cgroup related fix.

File: virttest/virsh.py (hunk 3 above)
Comment (lento-sun) [reply virttest/virsh.py]: the whole blkdeviotune() is not containing enough available virsh api params, will be updated in another pr.

Comment (Yingshun) [reply virttest/virsh.py]: It makes sense. LGTM.

File: virttest/virsh.py [hunk 4]
```diff
@@ -4209,10 +4209,18 @@ def vol_upload(name, dfile, options="", **dargs):
     return command(cmd, **dargs)
 
 
-def blkiotune(name, weight=None, device_weights=None, options=None, **dargs):
+def blkiotune(name, weight=None, device_weights=None, device_read_iops_sec=None,
```
Comment (dzhengfy) [inline virttest/virsh.py:4212]: I am not sure if we will have other more parameters need to be added in this function. It seems too long. Maybe a param_dict is more simple. But however, it is OK to me so far.

**Pull request merged**